Makefile

```diff
@@ -218,6 +234,12 @@ $(CRICTL):
 	mv crictl $(TOOLSDIR)/bin/crictl
 	chmod +x $(TOOLSDIR)/bin/crictl
 
+$(PROTOC):
```

Comment:
Maybe add the check-linux target as a dependency. Or fix binary url to work with other os-es.